docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -309,6 +309,102 @@
               description: 'File descriptors limit at {{ $labels.instance }} is currently at {{ printf "%.2f" $value }}%.',
             },
           },
+          {
+            alert: 'NodeCPUHighUsage',
```

Comment:
High CPU usage is not a problem and can just be an indicator or properly utilizing your machine, so I'd remove these

Reply:
perhaps, as long as we can alert on high system load(saturation).

Reply:
Yeah, CPU usage is good. :) I mean, this would be a case for the "info" level alerts that I like to promote, but I don't think we have them here in the mixin.

(Info level alerts notify nobody, but you could look at the alerts page while troubleshooting. They point to things that are not problems _per se_ and might be OK, but which you might be interested _while_ there is an actual incident happening.)

Reply:
Yeah I'd be fine with a 'info' level severity. No reason to now just introduce that now that we're on this.

Reply:
I'll make it an info according to this guideline:

```
info for alerts that do not require any action by itself but mark something as “out of the ordinary”. Those alerts aren’t usually routed anywhere, but can be inspected during troubleshooting.
```

Reply:
Perhaps a warning if the usage stays above 99% for 1h would be viable?  That would be a case where the host is at capacity and scheduling more tasks there would result in performance degradation.  It is a risk folks can accept but something that should be considered as part of the capacity plan.